crates/toml_edit/src/de/value.rs

```diff
@@ -8,14 +8,15 @@ use crate::de::TableDeserializer;
 /// Deserialization implementation for TOML [values][crate::Value].
 ///
 /// Can be created either directly from TOML strings, using [`std::str::FromStr`],
-/// or from parsed [values][crate::Value] using [`serde::de::IntoDeserializer::into_deserializer`].
+/// or from parsed [values][crate::Value] using
+/// [`IntoDeserializer::into_deserializer`][serde_core::de::IntoDeserializer::into_deserializer].
```

Comment:
Same Q here. This one I decided to handle slightly differently (shortening the displayed path). No strong opinion from me though.